### PR TITLE
Replace "www.example.com" with "fqdn" in MTV repo

### PR DESCRIPTION
**Pull request by RichardHoch**
MTV 2.2+

Resolves: https://bugzilla.redhat.com/show_bug.cgi?id=2067152

Replaces "www.example.com" in MTV documentation.

### CODE REVIEW

File: documentation/modules/retrieving-validation-service-json.adoc [hunk 1]
```diff
@@ -132,7 +132,7 @@ $ GET https://<inventory_service_route>/providers/<provider>/<UUID>/workloads/<v
                 "id": "domain-c26"
             },
             "revision": 1,
-            "name": "www.example.com",
+            "name": "fqdn",
```
[the remark this answers is not in the document]
Comment (mguetta1) [reply documentation/modules/retrieving-validation-service-json.adoc:135]: I am also not sure about this change.
@amastbau could you please confirm that?

[the remark this answers is not in the document]
Comment (RichardHoch) [reply documentation/modules/retrieving-validation-service-json.adoc:135]: @mguetta1 Did you get an answer?

[the remark this answers is not in the document]
Comment (mguetta1) [reply documentation/modules/retrieving-validation-service-json.adoc:135]: Not yet

**Comment by RichardHoch** [comment]
@istein1 I made the changes following your last comment in the BZ. Please review.

File: documentation/modules/creating-validation-rule.adoc [hunk 2]
```diff
@@ -121,7 +121,5 @@ spec:
 EOF
 ----
 <1> Allowed values are `ovirt` and `vsphere`.
-<2> Specify the API end point URL, for example, `https://<www.example.com>/sdk` for vSphere or `https://<www.example.com>/ovirt-engine/api/` for {rhv-short}.
-<3> Specify the name of provider `Secret` CR.
-
-You must update the rules version after creating a custom rule so that the `Inventory` service detects the changes and validates the VMs.
+<2> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere or `https://<RHV engine host>/ovirt-engine/api/` for {rhv-short}.
```
Comment (mguetta1) [inline documentation/modules/creating-validation-rule.adoc:124]: @RichardHoch please use {rhv-short}_engine_host

File: documentation/modules/creating-validation-rule.adoc [hunk 3]
```diff
@@ -121,7 +121,5 @@ spec:
...
+<3> Specify the name of the provider `Secret` CRHV engine host. You must update the rules version after creating a custom rule so that the `Inventory` service detects the changes and validates the VMs.
```
Comment (mguetta1) [inline documentation/modules/creating-validation-rule.adoc:125]: @RichardHoch was this changed by mistake?

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 4]
```diff
@@ -57,7 +57,7 @@ EOF
 ----
 <1> Specify the base64-encoded vCenter user or the {rhv-short} {manager} user.
 <2> Specify the base64-encoded password.
-<3> {rhv-short} only: Specify the base64-encoded CA certificate of the {manager}. You can retrieve it at `https://<www.example.com>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
+<3> {rhv-short} only: Specify the base64-encoded CA certificate of the {manager}. You can retrieve it at `https://<RHV engine host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA`.
```
Comment (mguetta1) [inline documentation/modules/migrating-virtual-machines-cli.adoc:60]: please use {rhv-short}_engine_host

File: documentation/modules/migrating-virtual-machines-cli.adoc [hunk 5]
```diff
@@ -79,7 +79,7 @@ spec:
 EOF
 ----
 <1> Allowed values are `ovirt` and `vsphere`.
-<2> Specify the API end point URL, for example, `https://<www.example.com>/sdk` for vSphere or `https://<www.example.com>/ovirt-engine/api/` for {rhv-short}.
+<2> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere or `https://<RHV engine host>/ovirt-engine/api/` for {rhv-short}.
```
Comment (mguetta1) [inline documentation/modules/migrating-virtual-machines-cli.adoc:82]: please use {rhv-short}_engine_host

File: documentation/modules/rhv-prerequisites.adoc [hunk 6]
```diff
@@ -10,4 +10,4 @@ The following prerequisites apply to {rhv-full} migrations:
 
 * You must have the CA certificate of the {manager}.
 +
-You can obtain the CA certificate by navigating to `https://<www.example.com>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA` in a browser.
+You can obtain the CA certificate by navigating to `https://<RHV engine host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA` in a browser.
```
Comment (mguetta1) [inline documentation/modules/rhv-prerequisites.adoc:13]: please use {rhv-short}_engine_host

File: documentation/modules/retrieving-validation-service-json.adoc [hunk 7]
```diff
@@ -132,7 +132,7 @@ $ GET https://<inventory_service_route>/providers/<provider>/<UUID>/workloads/<v
                 "id": "domain-c26"
             },
             "revision": 1,
-            "name": "www.example.com",
+            "name": "IP address or host name of the vCenter host or RHV Engine host",
```
Comment (mguetta1) [inline documentation/modules/retrieving-validation-service-json.adoc:135]: please use {rhv-short}

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: documentation/modules/creating-validation-rule.adoc (hunk 2 above)
Comment (RichardHoch) [reply documentation/modules/creating-validation-rule.adoc:124]: Done.

File: documentation/modules/creating-validation-rule.adoc (hunk 3 above)
Comment (RichardHoch) [reply documentation/modules/creating-validation-rule.adoc:125]: <2> is following Ilanit's comment in the BZ. Did i misunderstand.
<3. is a mistake; I reverted to the original correct text.

File: documentation/modules/migrating-virtual-machines-cli.adoc (hunk 4 above)
Comment (RichardHoch) [reply documentation/modules/migrating-virtual-machines-cli.adoc:60]: Done.

File: documentation/modules/migrating-virtual-machines-cli.adoc (hunk 5 above)
Comment (RichardHoch) [reply documentation/modules/migrating-virtual-machines-cli.adoc:82]: Done.

File: documentation/modules/rhv-prerequisites.adoc (hunk 6 above)
Comment (RichardHoch) [reply documentation/modules/rhv-prerequisites.adoc:13]: Done.

File: documentation/modules/retrieving-validation-service-json.adoc (hunk 7 above)
Comment (RichardHoch) [reply documentation/modules/retrieving-validation-service-json.adoc:135]: Done.

File: documentation/modules/creating-validation-rule.adoc (hunk 3 above)
Comment (mguetta1) [reply documentation/modules/creating-validation-rule.adoc:125]: This question was for <3>, thanks.

File: documentation/modules/creating-validation-rule.adoc [hunk 8]
```diff
@@ -121,7 +121,7 @@ spec:
 EOF
 ----
 <1> Allowed values are `ovirt` and `vsphere`.
-<2> Specify the API end point URL, for example, `https://<www.example.com>/sdk` for vSphere or `https://<www.example.com>/ovirt-engine/api/` for {rhv-short}.
-<3> Specify the name of provider `Secret` CR.
-
+<2> Specify the API end point URL, for example, `https://<vCenter_host>/sdk` for vSphere or `https://<{rhv-short}_engine_host>/ovirt-engine/api/` for {rhv-short}.
+<3> Specify the name of the provider `Secret` CR. 
++
```
Comment (mguetta1) [inline documentation/modules/creating-validation-rule.adoc:126]: @RichardHoch I don't know what is the effect of this addition (+), do we need it?

**Review by mguetta1 — CHANGES_REQUESTED** [review]

File: documentation/modules/creating-validation-rule.adoc (hunk 8 above)
Comment (RichardHoch) [reply documentation/modules/creating-validation-rule.adoc:126]: Was not needed; removed

Comment (RichardHoch) [reply documentation/modules/creating-validation-rule.adoc]: Removed.

**Review by mguetta1 — APPROVED** [review]
LGTM, thanks

**Pull request merged**